tests/test_git/test_from_git.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for creating a griffe Module from specific commits in a git repository."""
```

Comment:
Since there's only one test file for now (and if you move the `_repo` to the `fixtures` directory as mentioned above), you can move this up one directory, and rename it `test_git.py` since that's the module name in Griffe sources.